.github/workflows/lighthouse.yml

```
@@ -0,0 +1,30 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+          
+      - name: Install dependencies
+        run: cd frontend && npm install
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)